tools/wpt/revlist.py

```diff
@@ -0,0 +1,128 @@
+    if value.lower() == "twelve-hourly":
+        return 43200
+    if value.lower() == "daily":
+        return 86400
+    if value.lower() == "weekly":
+        return 604800
+    return int(value)
+
+
+def get_tagged_revisions(pattern):
+    # type: (bytes) -> List[..., Dict]
+    '''
+    Returns the tagged revisions indexed by the committer date.
+    '''
+    git = get_git_cmd(wpt_root)
+    args = [
+        pattern,
+        '--sort=-committerdate',
+        '--format=%(refname:lstrip=2) %(objectname) %(committerdate:raw)',
+        '--count=100000'
+    ]
+    for tagged_revision in git("for-each-ref", *args).split("\n"):
+        revision_tag, revision_commit, revision_date, _ = tagged_revision.split(" ")  # noqa E501
+        revision_date = int(revision_date)
+        yield revision_tag, revision_commit, revision_date
+
+
+def list_tagged_revisons(epoch, max_count):
+    # type: (**Any) -> List[Text]
+    logger.debug("list_tagged_revisons(%s, %s)" % (epoch, max_count))
+    # Set an offset to start to count the the weekly epoch from
+    # Monday 00:00:00. This is particularly important for the weekly epoch
+    # because fix the start of the epoch to Monday. This offset is calculated
+    # from Thursday, 1 January 1970 0:00:00 to Monday, 29 December 1969 0:00:00
+    epoch_offset = -259200
+    # "epoch_threshold" set a safety margin after this time it is fine to
+    # consider that any tags are created and pushed.
+    epoch_threshold = 600
+    epoch_until = calendar.timegm(time.gmtime()) - epoch_threshold
+    count = 0
+
+    # Iterates the tagged revisions in descending order finding the more
+    # recent commit still older than a "min_timestamp" value.
+    # When a commit is found "min_timestamp" is set to a new value multiplier of
+    # "epoch" but still below of the date of the current commit found.
+    # This needed to deal with intervals where no candidates were found
+    # for the current "epoch" and the next candidate found is yet below
+    # the lower values of the interval (it is the case of J and I for the
+    # interval between Wed and Tue, in the example). The algorithm fix
+    # the next "min_timestamp" value based on the date value of the current one
+    # skipping the intermediate values.
+    # The loop ends once we reached the required number of revisions to return
+    # or the are no more tagged revisions or the min_timestamp reach zero.
+    #
+    #   Fri   Sat   Sun   Mon   Tue   Wed   Thu   Fri   Sat
+    #    |     |     |     |     |     |     |     |     |
+    # -A---B-C---DEF---G---H--IJ----------K-----L-M----N--O--
+    #                                                       ^
+    #                                                      now
+    # Expected result: N,M,K,J,H,G,F,C,A
+
+    min_timestamp = calculate_min_timestamp(epoch_until, epoch, epoch_offset)
+    for revision_tag, revision_commit, revision_date in get_tagged_revisions("refs/tags/merge_pr_*"):  # noqa E501
```

Comment:
```suggestion
    for tag, commit, date in get_tagged_revisions("refs/tags/merge_pr_*"):
```

The `revision_` prefix isn't necessary, and removing it makes the line short enough :)